postpic/datareader/datareader.py

```diff
@@ -266,15 +266,31 @@ def name(self):
     def name(self, val):
         self._name = str(val) if bool(val) else None
 
-    def __str__(self):
-        return '<Dumpreader initialized with "' \
-               + str(self.dumpidentifier) + '">'
+    def __repr__(self):
+        return '<Dumpreader at "{:}">'.format(str(self.dumpidentifier))
 
     def __eq__(self, other):
         """
         two dumpreader are equal, if they represent the same dump.
+
+        Assuming the dumpidentifier are paths to the dumpfiles, his may give a "False",
```

Comment:
Assuming the dumpidentifier **is a** path to **a** dumpfile,  **simple string comparison** may give a "False"....